scripts/py_matter_idl/examples/matter_idl_plugin/__init__.py

```diff
@@ -0,0 +1,326 @@
+            return "uint32"
+        case "devtype_id":
+            return "uint32"
+        case "fabric_idx":
+            return "uint32"
+        case "vendor_id":
+            return "uint32"
+        case "status_code":
+            return "uint32"
+        case "faulttype":
+            return "uint32"
+        case 'epoch_s':
+            return "uint32"
+        case "levelcontroloptions":
+            return "uint32"
+        case "percent100ths":
+            return "uint32"
+        case "percent":
+            return "uint32"
+
+        case "int8u":
+            return "uint32"
+        case "int16u":
+            return "uint32"
+        case "int24u":
+            return "uint32"
+        case "int32u":
+            return "uint32"
+
+        case "enum64":
+            return "uint64"
+        case "bitmap64":
+            return "uint64"
+        case "node_id":
+            return "uint64"
+        case "fabric_id":
+            return "uint64"
+        case "epoch_us":
+            return "uint64"
+        case "int40u":
+            return "uint64"
+        case "int48u":
+            return "uint64"
+        case "int56u":
+            return "uint64"
+        case "int64u":
+            return "uint64"
+
+        case "int8s":
+            return "int32"
+        case "int16s":
+            return "int32"
+        case "int24s":
+            return "int32"
+        case "int32s":
+            return "int32"
+
+        case "int40s":
+            return "int64"
+        case "int48s":
+            return "int64"
+        case "int56s":
+            return "int64"
+        case "int64s":
+            return "int64"
+
+        case "long_char_string":
+            return "string"
+        case "char_string":
+            return "string"
+
+        case "long_octet_string":
+            return "bytes"
+        case "octet_string":
+            return "bytes"
+
+        case "single":
+            return "float"
+        case "float":
+            return "float"
+        case "double":
+            return "double"
+
+    return s
+
+
+# Python enum of types
+class DataType:
+    UINT = 1
+    INT = 2
+    BOOL = 3
+    CHAR_STRING = 4
+    OCT_STRING = 5
+    STRUCT = 6
+    FLOAT = 7
+    DOUBLE = 8
+
+
+def typeToTypeNum(s):
+    match(s):
+        case "uint32": return DataType.UINT
+        case "uint64": return DataType.UINT
+        case "int32": return DataType.INT
+        case "int64": return DataType.INT
+        case "bool": return DataType.BOOL
+        case "string": return DataType.CHAR_STRING
+        case "bytes": return DataType.OCT_STRING
+        case "float": return DataType.FLOAT
+        case "double": return DataType.DOUBLE
+
+    return DataType.STRUCT
+
+
+def commandArgs(command: Command, cluster: Cluster):
+    # Find command.input_arg in cluster.structs
+    for struct in cluster.structs:
+        if struct.name == command.input_param:
+            return struct.fields
+
+    return []
```

Comment:
should we log when something is not found or is that expected in general? Same comment for methods below.